_pages/jlacomis.yaml

```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: jlacomis
+pageUrl: https://github.com/UVaCS4720/github-102-desktop-s17-jlacomis/pull/1
```

Comment:
@jlacomis the page url should be `yourusername.github.io` if you followed the instructions. Let us know if you need any help correcting that